frontend/lib/ui/home_page/home_page_provider.dart

```diff
@@ -1,17 +1,16 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
-import 'package:frontend/ui/home_page/home_page_bloc.dart';
-import 'package:frontend/ui/home_page/home_page_view.dart';
+
+import 'home_page_bloc.dart';
+import 'home_page_view.dart';
 
 class HomePageProvider extends StatelessWidget {
```

Comment:
create providers like this.no need to use widgets.

![image](https://github.com/fcodelabs/training-ramp-up/assets/47570687/f25e5736-f324-490d-9ba8-b080099b61c5)